tests/test_rpc_cli.rs

```diff
@@ -43,7 +43,7 @@ fn test_service_ready() -> bool {
     let mut client_cfg = devnet::ClientConfig::default();
     client_cfg.base_path = base_path;
     client_cfg.log_level = log::LevelFilter::Off;
-    thread::spawn(|| { devnet::start_devnet(Some(network_cfg), Some(client_cfg)) });
+    thread::spawn(|| { devnet::start_devnet(Some(network_cfg), Some(client_cfg), None) });
```

Comment:
we should pass in a `DevnetConfig` and configure it to be lower .. ~1-5ms?

Reply:
Fixed in bb1ba6b